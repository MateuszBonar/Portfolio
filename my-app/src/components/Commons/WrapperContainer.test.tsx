import { WrapperContainer } from 'Components';
import { render } from '@testing-library/react';


const mocks = {
  classes: ['about', 'container'],
  id: 'about',
}

describe('WrapperContainer', () => {
  it('renders correctly', () => {
    render(<WrapperContainer classes={mocks.classes} id={mocks.id} />);
  })

  it('renders about', () => {
    const { container } = render(<WrapperContainer classes={mocks.classes} id={mocks.id} />);
    expect(container).toMatchSnapshot();
  });
})