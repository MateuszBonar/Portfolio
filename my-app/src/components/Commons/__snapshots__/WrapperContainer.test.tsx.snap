// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WrapperContainer renders about 1`] = `
<div>
  <div
    class="about"
    id="about"
  >
    <div
      class="container"
    />
  </div>
</div>
`;

exports[`WrapperContainer renders correctly 1`] = `
<div>
  <div
    class="about"
    id="about"
  >
    <div
      class="container"
    />
  </div>
</div>
`;
